### PR TITLE
adding RQ3 description and table

### PR DESCRIPTION
**Pull request by Keyshal-211**

### CODE REVIEW

File: paper/user_study.tex [hunk 1]
```diff
@@ -338,13 +338,35 @@ \subsubsection{\textbf{RQ2:} Are users able to perform the followings types of t
 
 
 \subsubsection{\textbf{RQ3:} Would people use the tool in practice?} % Keyuri
+Table~\ref{table:results} shows the individual opinions regarding practical use, their thoughts on the fields in which this tool could be used and the target users for using this tool.
```
Comment (PaulBredl) [inline paper/user_study.tex:341]: ref does not match with the label in line 246

File: paper/user_study.tex [hunk 2]
```diff
@@ -338,13 +338,35 @@ \subsubsection{\textbf{RQ2:} Are users able to perform the followings types of t
...
 
+    \centering
+    \caption{Use in Practice} \label{table:user's opinion about use of tool}
+    \begin{tabular}{p{0.3\linewidth}p{0.3\linewidth}p{0.3\linewidth}}
+        \toprule
+        \thead{} &
+        \thead{Work Field / Profession} & \thead{Opinion regarding Practical Use} \\
+        \midrule
+        User Study 1 & Professor &
+        Once the schema is loaded then one can start easily and after doing one time use, it will be easy to understand the 3 views. \\
```
Comment (PaulBredl) [inline paper/user_study.tex:353]: It is not clear what is a direct citation and what is paraphrased.
We could even make it a bit simpler and make a very simple table here where one column would be "Would you use this tool yourself?" and one column would be "Would you recommend this tool to others?" 
And then only yes/no answers (or no data)
Then we could also have a numerical value we can refer to (4/5 pariticipants would recommend this tool to others, just as an example, not having checked the data)

@Logende would do you think?

Comment (Logende) [reply paper/user_study.tex:353]: I think this is the status without quotations, everything is paraphrased. I like your idea regarding the boolean values.
And I think we could still have those quotations. And they should also be in quotation marks then and match word by word what the participants said (the most relevant sections).

**Comment by PaulBredl** [comment]
@Keyshal-211 can you solve the merge conflicts?

**Pull request closed without merging**